File: CP022/Java/Main.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        for (int i = n; i >=1 ; i--) { // for down to
            for (int j = 1; j <= i; j++) {
                System.out.print("* ");
            }
            System.out.println();
        }
    }
}

File: CP022/C++/Main.cpp
#include <iostream>
using namespace std;
int main() {
    int n;
    cin >> n;
    for(int i=n;i>=1;i--){//for down to
        for(int j = 1;j<=i;j++){
            cout << "* ";
        }
        cout << endl;
    }
    return 0;
}
